src/state/stake/useFarmContractsForVersion.ts

```diff
@@ -47,20 +49,26 @@ export function useFarmContractsForVersion(chefVersion: ChefVersions): StakingTr
 
     return lpAddresses.map((lpAddress, index) => {
       // User based info
+
       const userStaked = userInfo[index]
       const activeFarmID = getActiveFarmID(lpAddress)
+
+      const stablePoolFarm = activeFarms[activeFarmID].stableSwapPoolName
+      const stablePool = stablePoolFarm ? STABLESWAP_POOLS[stablePoolFarm] : null
```

Comment:
```suggestion
      const stablePool = STABLESWAP_POOLS?.[stablePoolFarm]
```

for educational purposes

Reply:
yeah, but the problem is that:

```
Type 'null' cannot be used as an index type
``` 
there may be a way though

Reply:
```suggestion
      const stablePool = stablePoolFarm && STABLESWAP_POOLS?.[stablePoolFarm]
```

Reply:
lol nah that would actually evaluate as false too kek

Reply:
rrrreeeeeee

Reply:
LOL right, covid getting into my mind sorry, pushing and merging

Reply:
> lol nah that would actually evaluate as false too kek

works! :)